### DISCUSSION TITLE
Unable to select multiple panels when using 'Dashboard' as a datasource

### DISCUSSION THREAD

**Discussion by dancernogorsky** [grafana/grafana#52303] [Dashboards, Panels & Visualizations]
**What happened**: 
Unable to select multiple panels when using 'Dashboard' as a datasource.

**How to reproduce it (as minimally and precisely as possible)**:
- Create a dashboard with multiple panels.
- Create a panel and select 'Dashboard' as the datasource. You will only be able to add one (no duplicate option as well). 

**Environment**:
- Grafana version: 9.0.2

![image](https://user-images.githubusercontent.com/29049683/178317914-699dd1b2-3745-40e0-8c28-124cdec5482f.png)

![image](https://user-images.githubusercontent.com/29049683/178317975-c095e8c1-a2c5-48b8-b22e-5f70fd52c68a.png)

**Comment by mellieA** [comment]
@dancernogorsky this looks like a feature request so I'll move it over to github discussions for our dev team to take a look at. 

I see what you mean, but it took me a few tries to understand (correct me if I'm still missing something). My understanding is that you want to be able to: 

* create panel 1
* create panel 2
* Feature request: 
  * Add ability to use *both* panel 1 & panel 2 as sources in panel 3. For example, as a mixed datasource where you can select two different sources for a single panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
